GoogleForCreators/web-stories-wp
# Dashboard: Sort Dropdown for views

# Overview 
Adds sort dropdown UI to my stories and templates as a shared component in views
Functionality is hooked up for my stories but not templates, but it's prepped for whenever templates api is ready for action!  

- gutted the lisBarContainer in views/shared - absorbed it into bodyViewOptions 

## Testing 
- change the sort on my stories and see the view respond. I'm relying on the api for the actual sort, but it should sync as follows: 
  - sort by name = ordered ascending (a - z)
  - sort by date created = ordered descending (most recent to least recent)
  - sort by last modified = ordered descending  (most recent to least recent)
  - sort by last opened = ordered descending  (most recent to least recent)
  - sort by created by  = ordered ascending (a -z)
- Resize screen, on < 684px see the view toggle and the sort stack

## Screenshot
<img width="408" alt="Screen Shot 2020-04-10 at 3 38 29 PM" src="https://user-images.githubusercontent.com/10720454/79027511-59ecc000-7b41-11ea-8334-33e9021bcf18.png">


## Open questions 
- What is the difference between last modified and last opened? I can't find any last opened data in stories response. Am currently using modified for last opened until can sort this out. Didn't realize this until friday afternoon. So will ask in slack about it come Monday. Don't want it to be buried and forgotten over the weekend and don't feel it's a big enough issue to block this PR. 
- We need to customize the 'created by' to first put the current user's stories/templates and THEN everyone elses. This would be a custom orderBy w/ wordpress's api.
